docs/Changelog.md

```diff
@@ -8024,6 +7996,92 @@ This version of the operator has been available since version 7 of the default O
 </dl>
 
 ## Version 8 of the default ONNX operator set
+### <a name="Conv_Integer-8"></a>**Conv_Integer-8**</a>
+
+  The integer convolution operator consumes an input tensor, a filter and a padding value, and computes the output. The production MUST never overflow. The accumulation may overflow if and only if in 32 bits.
```

Comment:
If inputs are 16 bits wide, the product will, generally, overflow. We should either remove 16-bit input types, specify overflow behavior, or specify 64-bit output for 16-bit inputs.

Reply:
Let's remove 16 bits for now then.